apps/alphabet_mobile/SETUP_README.md

```diff
@@ -0,0 +1,65 @@
+# Create a React Native project on Ubuntu 22.04
+
+This is a guide to creating a React Native project and setting up the environment to support Android Studio's emulator.
+
+## Prerequisites
+
+You'll need to install the Android emulator.
+
+### Install Android Studio
+
+https://developer.android.com/studio
+
+### Create a directory to hold the Android Studio file
+
+> > mkdir opt
+
+### Move Android Studio to the opt directory in home
```

Comment:
These headings are clearly sub-headings of "Install Android Studio". Let's use proper semantic markup here. 
#### Configure Environment Variables